### PR TITLE
presets: Add a RHEL 7.4 preset

### PR DESCRIPTION
**Pull request by fabiand**
Signed-off-by: Fabian Deutsch <fabiand@fedoraproject.org>

### CODE REVIEW

**Comment by fabiand** [comment]
Why actually?

File: cluster/examples/vm-template-rhel7.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ metadata:
     tags: kubevirt,ocp,template,linux,virtualmachine
   creationTimestamp: null
   labels:
-    kubevirt.io/os: rhel-7.4
+    kubevirt.io/os: rhel7.4
```
Comment (itamarh) [inline cluster/examples/vm-template-rhel7.yaml]: why is the preset for a specific minor version vs. just rhel7?

File: manifests/release/demo-content.yaml.in [hunk 2]
```diff
@@ -2,7 +2,7 @@
 apiVersion: kubevirt.io/v1alpha2
 kind: VirtualMachineInstancePreset
 metadata:
-  name: windows-server-2012r2
+  name: microsoft-windows-server-2012r2
```
Comment (itamarh) [inline manifests/release/demo-content.yaml.in]: just a nit seems unrelated to a patch titled "rhel", but nm.

File: cluster/examples/vm-template-rhel7.yaml (hunk 1 above)
Comment (fabiand) [reply cluster/examples/vm-template-rhel7.yaml]: I meant to answer that earlier.

`kubevirt.io/os` takes a libosinfo identifier as a value. And there minor versions have identifiers, not the major version.

**Comment by fabiand** [comment]
I'm suggesting to keep presets and templates in a repository separate from KubeVirt.

I've started this repo https://github.com/fabiand/common-templates to gather presets and templates for common guests., including functional tests for templates and presets.

**Pull request closed without merging**